profiles/pci/graphic_drivers/profiles.toml

```diff
@@ -189,9 +189,14 @@ post_install = """
         # Add libva-nvidia-driver to profile
         echo "export LIBVA_DRIVER_NAME=nvidia" > /etc/profile.d/nvidia-vaapi.sh
     fi
+    mkdir -p /etc/sddm.conf.d
+    cat << EOF >/etc/sddm.conf.d/01-wayland.conf
+# THIS IS A STUB GENERATED BY CHWD TO OVERRIDE THE CACHYOS-KDE-SETTINGS PACKAGE, DO NOT TOUCH.
```

Comment:
shouldn't it be that then. to overwrite the display server used
```
[General]
DisplayServer=x11
```

as it will read, and load nothing. I could be wrong

Reply:
Valid concern, fixed, thanks.